README.md

```diff
@@ -12,28 +15,39 @@
 </div>
 </div>
 
+<!-- ---------- Description ---------- -->
 ## Features
 
 - [x] 30 supported languages
 - [x] Material Design 3 (You)
 - [x] Dark and light theme
 - [x] Entirely Open Source
 - [X] Written in Jetpack Compose 
+- [X] Support native Android sharing system
+- [X] Multilingual interface
 
+<!-- ---------- Download ---------- -->
 ## Download
 
 [<img src="https://gitlab.com/IzzyOnDroid/repo/-/raw/master/assets/IzzyOnDroid.png" alt="Get it on IzzyOnDroid" height="80">](https://apt.izzysoft.de/fdroid/index/apk/com.bnyro.translate)
 [<img src="ghbadge.png" alt="Get it on GitHub" height="80">](https://github.com/bnyro/translateyou/releases)
 
-## Different versions
-There are currently two different versions available:
+<!-- ---------- App Versions ---------- -->
+## Versions
+**There are currently two different versions of TY available:**
 
-#### Translate You Libre
+### Translate You Libre
 Only supports the <a href="https://github.com/LibreTranslate/LibreTranslate">LibreTranslate API</a> and therefore is fully libre.
 
-#### Translate You Plus
-Additionally supports <a href="https://github.com/thedaviddelta/lingva-translate">LingvaTranslate</a> which uses Google Translate to fetch translations and thus owns the NonFreeNet anti-feature.
+### Translate You Plus
+[**NonFreeNet anti-feature**](https://f-droid.org/docs/Anti-Features/#NonFreeNet)
```

Comment:
I'd change that to `[Anti-feature: NonFreeNet](...), because it supports translation engines that are not open source`.

Reply:
would this version have your approval:
[...](...) Due to supports translation engines that are not open source.

Reply:
> would this version have your approval: [...](...) Due to supports translation engines that are not open source.

Yes, like that one! :)